utils/mil_tools/nodes/online_bagger.py

```diff
@@ -146,15 +167,15 @@ def subscribe(self):
         Return number of topics that failed subscription
         """
 
-        for topic in self.subscriber_list:
+        for i, topic in enumerate(self.subscriber_list):
             if not topic[2]:
                 msg_class = rostopic.get_topic_class(topic[0])
                 if msg_class[1] is not None:
-                    self.successful_subscription_count = self.successful_subscription_count + 1
+                    self.successful_subscription_count += 1
                     rospy.Subscriber(topic[0], msg_class[0],
                                      lambda msg, _topic=topic[0]: self.bagger_callback(msg, _topic))
 
-                    topic[2] = True  # successful subscription
+                    self.subscriber_list[i] = (topic[0], topic[1], True)
```

Comment:
Yeah that would make things easier, [can do](https://www.youtube.com/watch?v=y9IcGZZnC1k)